docs/.templates/integration/schema.json

```
@@ -0,0 +1,84 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "Netdata Integrations file",
+    "properties": {
+        "categories_list": {
+            "type": "object",
+            "description": "A list defining all the available categories for the integrations",
+            "properties": {
+                "list": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "name": {
+                                "type": "string",
+                                "description": "Name of the category"
+                            },
+                            "priority": {
+                                "type": "string",
+                                "description": "Priority of the category"
+                            },
+                            "id": {
+                                "type": "string",
+                                "description": "ID of the category that will be used by integrations"
+                            },
+                            "parent": {
+                                "type": [
+                                    "null",
+                                    "string"
+                                ],
+                                "description": "The category's parent ID, make it null if no parent"
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "integrations": {
+            "type": "array",
+            "description": "A list of integration elements, combination of metadata.yaml and other sources",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "integrations_name": {
+                        "type": "string"
+                    },
+                    "category": {
```

```suggestion
                    "category_id": {
```

Maybe `category_id` since it is "The ID for this category"?